### PR TITLE
groupData and briefing changes for #254 and #248

### PR DESCRIPTION
**Pull request by NikkoJT**
This PR completes #248 by adding civilian handling and group data to the groupData files, as well as adding group variable names to the civilian units.

It also partially completes #254 by adding handling for the NATO Woodland, Spetsnaz, and LDF factions to the groupData files, as well as group data for the Spetsnaz and LDF factions.

e: I don't expect another F3 release before Contact, but it is safe to add the Contact faction data before then anyway. It just won't do anything.

e: this PR now also includes briefings for Spetsnaz, LDF, and Looters, and group data for Looters.

### CODE REVIEW

File: f/briefing/fn_createBriefing.sqf [hunk 1]
```diff
@@ -49,7 +49,7 @@ if (serverCommandAvailable "#kick" || !isMultiplayer) then {
 // The following code blocks include faction-specific briefing files.
 
 // BLUFOR > NATO
-if (_unitfaction in ["blu_f","blu_t_f"]) exitwith {
+if (_unitfaction in ["blu_f","blu_t_f","blu_w_f"]) exitwith {
```
Comment (shadow-fa) [inline f/briefing/fn_createBriefing.sqf]: Was this on purpose?

**Comment by shadow-fa** [comment]
Do you also want to add the template units (in editor) as part of this PR?

**Comment by NikkoJT** [comment]
> Do you also want to add the template units (in editor) for the new factions as part of this PR?

Didn't intend to, as that's not been finalised yet. I'd prefer to keep this (relatively safe) change separate from the heavy part with a bigger risk of errors. Just for tidiness.

File: f/briefing/fn_createBriefing.sqf [hunk 2]
```diff
@@ -72,18 +72,37 @@ if (_unitfaction in ["opf_f","opf_t_f"]) exitwith {
...
+	[_unitfaction] call _fnc_debug;
+};
+
 // INDEPENDENT > SYNDIKAT
 if (_unitfaction in ["ind_c_f"]) exitwith {
 	#include "f_briefing_syndikat.sqf"
 	[_unitfaction] call _fnc_debug;
 };
 
+// INDEPENDENT > Looters
```
Comment (SamLex) [inline f/briefing/fn_createBriefing.sqf]: Might be good to say 'NPR (Looters)' to make the distinction between our name and the ingame name clear

File: f/groupMarkers/f_setLocalGroupMarkers.sqf [hunk 3]
```diff
@@ -67,6 +77,16 @@ if (_unitfaction in ["3ifb"]) then {
 	_groups = f_var_groupData_indfor_3ifb;
 };
 
+// Markers seen by players in 3IFB (virtual) slots.
```
Comment (SamLex) [inline f/groupMarkers/f_setLocalGroupMarkers.sqf]: Comment not updated to mention NPR/Looters

**Pull request merged**